cloud/amazon/ec2_elb_lb.py

```diff
@@ -698,6 +710,28 @@ def _set_cross_az_load_balancing(self):
         self.elb_conn.modify_lb_attribute(self.name, 'CrossZoneLoadBalancing',
                                           attributes.cross_zone_load_balancing.enabled)
 
+    def _set_access_log(self):
+        attributes = self.elb.get_attributes()
+        if self.access_logs is not None:
+            access_log = boto.ec2.elb.attributes.AccessLogAttribute()
+            access_log.enabled = True
+            access_log.s3_bucket_prefix = ''
+            access_log.emit_interval = 60
```

Comment:
Does this value need to be set explicitly here? I see from the AWS docs that the default is already 60.